ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/mac-security.j2

```diff
@@ -42,6 +43,17 @@ Keys:
 {%                     endif %}
 {%                 endfor %}
 {%             endif %}
+
+L2 Protocols:
+
+{%             if profile.l2_protocols | arista.avd.default([]) | length > 0 %}
```

Comment:
Move this `if` to right after line 45, so you include the "L2 Protocols:" _and_ all blank lines.

Reply:
Done